### DISCUSSION TITLE
How to post data to IDS Connector using http?

### DISCUSSION THREAD

**Discussion by YogarajRethinasamy** [International-Data-Spaces-Association/DataspaceConnector#82] [Q&A] [answered]
how to post data to IDS Connector using http

https://docs.aws.amazon.com/firehose/latest/dev/httpdeliveryrequestresponse.html

i want to send from firehose to IDS using http is that possible. 

IF possible means how to do that

**Comment by juliapampus** [comment, marked as answer]
Hi,

how to add local or remote data to the connector as a data provider is explained [here](https://international-data-spaces-association.github.io/DataspaceConnector/CommunicationGuide/v6/Provider#step-2-add-local-data). You could either write some service that pushes the data from a custom backend into the Dataspace Connector's database, or add retrieval information to an artifact so the connector is able to fetch the data on a request.
